### PR TITLE
Bitcoin deposit fees

### PR DESCRIPTION
**Pull request by fadeev**

### CODE REVIEW

File: docs/developers/omnichain/zrc-20.md [hunk 1]
```diff
@@ -65,6 +65,17 @@ correspond to a contract address on ZetaChain, the token will be deposited to
...
 
+When depositing native gas tokens from EVM-based connected chains, there is no
+additional cross-chain fee. If you send 1 token to a TSS address, you will
+receive 1 ZRC-20 version of the same token on ZetaChain.
+
+For Bitcoin deposits, which utilize the UTXO (Unspent Transaction Output) model,
+the process incurs additional fees. Unlike EVM-based chains, each deposited
+Bitcoin output incurs a fee when it is spent. To address this, both the
+depositor and the withdrawer share the cost of the spend. This fee is charged in
+advance as a deposit fee. Currently, the rate is set at 20 satoshis per virtual
```
Comment (brewmaster012) [inline docs/developers/omnichain/zrc-20.md:76]: @ws4charlie whats the planned change? Probably a good idea to put here as well.

Comment (ws4charlie) [reply docs/developers/omnichain/zrc-20.md:76]: Yeah. The problem of a fixed depositor rate `20 satoshis/vB` is described here:
https://github.com/zeta-chain/node/issues/1698

The planed PR: https://github.com/zeta-chain/node/pull/1747

The planned changed is
-  Given a inbound bitcoin tx contained in block X.
- The depositor fee will be `Avg-Fee-Rate-Block-X * GasPriceMultiplier * Deposit-incurred-vBytes`. The `Deposit-incurred-vBytes` is fixed as `68vB` atm, and the `GasPriceMultiplier` is defaulted to `2` atm.

Comment (fadeev) [reply docs/developers/omnichain/zrc-20.md]: 8662c89

**Review by ws4charlie — APPROVED** [review]
looks good

**Pull request merged**